Dockerfile

```diff
@@ -1,6 +1,7 @@
 # syntax = docker/dockerfile:experimental
 
 # Copyright (C) 2020 Bosch Software Innovations GmbH
+# Copyright (C) 2021 Alliander N.V.
```

Comment:
Commit message:

> Related issue: #3969

Please slightly change the syntax to match https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword so that the issue gets properly linked / auto-closed when the PR is merged.

Reply:
Check, I just did. Good to keep in mind for future commits, thanks.